registry/server/api_topics.go

```diff
@@ -30,6 +31,8 @@ var (
 	ErrInsufficientBrokers = status.Error(codes.FailedPrecondition, "insufficient number of brokers")
 	// ErrInvalidBrokerId error.
 	ErrInvalidBrokerId = status.Error(codes.FailedPrecondition, "invalid broker id")
+	// ErrTaggingTopicTimedOut
+	ErrTaggingTopicTimedOut = status.Error(codes.Internal, "tagging topic timed out")
```

Comment:
I think there's a `DeadlineExceeded` or similar code that seems to fit better here.

Reply:
It's kind of a dead error because the only time we'll see this, grpc will have already automatically surfaced the context timeout errors. I'll have to check if there's convention on chaining RPCs for timeout errors in this case.